2/scss/big-slaider.scss

```diff
@@ -0,0 +1,107 @@
+@import "vars";
+@import "fonts";
+@import "mixins";
+
+.wrapper-big-slaider {
+  margin-top: 30px;
+  padding-top: 30px;
+  background: linear-gradient(15deg, #e0e0e0, #ffffff);
+  .big-slider-container {
+    min-height: 1085px;
+    max-width: 1440px;
+    margin: auto;
+    .slider-title-container {
+      text-align: center;
+      .slider-title{
+        @include text-size-standart;
+      }
+    }
+  }
+}
+
+.inner-wrapper {
+  .container-dots {
+    margin: 30px 0 30px 0;
+    .slick-dots {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      li {
+        margin: 0 10px 0 10px;
+        list-style: none;
+      }
+      button {
+        cursor: pointer;
+        padding: 5px;
+        position: relative;
+        font-size: 0;
+        border: 0;
+        background-color: transparent;
+        &::before {
+          position: absolute;
+          content: "";
+          width: 6px;
+          height: 6px;
+          background-color: $colorTextlinks;
+          border-radius: 20px;
+          left: 2px;
+          top: 2px;
+        }
+      }
+      .slick-active {
+        button {
+          margin-top: 8px;
+          border: 1px solid $colorTextlinks;
+          width: 24px;
+          height: 24px;
+          border-radius: 20px;
+          &::before {
+            left: 8px;
+            top: 8px;
```

Comment:
you can position like this:
```
left: 50%;
top: 50%;
transform: translate(-50%, -50%);
```
This is universal code that will work all the time, no matter what element's and parent's size

Reply:
done